config/redis.yml

```diff
@@ -59,6 +59,9 @@ development: &defaults
     threshold_ttl: 86400
     count_limit: 30
     count_ttl: 604800
+  user_account_details:
+    namespace: user-account-details
+    each_ttl: 1296000
```

Comment:
not a big deal, but I tend to put inline comments with large amount of seconds with the human readable number to save people some math in the future :)

Reply:
Good call!